amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -323,14 +322,14 @@ export class TableDetail extends React.Component<TableDetailProps & RouteCompone
     } else if (this.state.statusCode === 500) {
       innerContent = (
         <div className="container error-label">
-          <Breadcrumb path='/' text='Search Results'/>
+          <Breadcrumb path="/" text="Home" />
```

Comment:
Was this a find/replace mistake? On the `TableDetail` page we want the logic of whether or not to go back to `Home` or `SearchPage` to be executed.